myRental/migrations/20180225201038_add_information_to_user.js

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
Get rid of this newline

Reply:
After inspection of the files on my PC the two migration files look normal.